application/templates/forms/_text_input.html

```diff
@@ -0,0 +1,22 @@
+{# Required parameters: field, class_, type, disabled, textarea, diffs, field_params #}
```

Comment:
`type` doesn't sem to be used here either?

Reply:
You're right, I think it's slurped into `field_params` now. Will update.